Meadowlark-js/Dockerfile

```diff
@@ -11,7 +11,10 @@ RUN apt-get update && \
   # dumb-init is useful for running node properly (not as PID 1)
   apt-get install -y --no-install-recommends dumb-init=1.2.5-1 && \
   apt-get clean && \
-  rm -rf /var/lib/apt/lists/*
+  rm -rf /var/lib/apt/lists/* && \
+  # create meadowlark.log file and give permissions to allow winston to write if enabled
+  touch /var/log/meadowlark.log && \
+  chmod 777 /var/log/meadowlark.log
```

Comment:
> **chmod 777**
> This means that owner, group and everyone has all the rights, i.e. to read, write and execute. This is a dangerous permission to have on any file and you should avoid using it.

Source: https://linuxhandbook.com/chmod-calculator/

I agree with that sentiment.

Since our Dockerfile has `USER node` in it, we are running under the user `node`. Therefore we should change the file's owner to `node`, and only give read & write permission to that user.

```bash
chown node /var/log/meadowlark.log && \
chmod 600  /var/log/meadowlark.log
```